hawc/apps/assessment/models.py

```diff
@@ -1038,7 +1057,7 @@ def rendered_page(
 
 
 reversion.register(DSSTox)
-reversion.register(Assessment)
+reversion.register(Assessment, follow=["values"])
```

Comment:
instead of adding this model to follow, add another row to just register this one by itself.